### PR TITLE
Add In-Person Payments Card Acquisition Flow

### PR DESCRIPTION
**Pull request by Kwok-he-Chu**
**Description**
* Added an advanced use case to check cardholder details before applying discounts or sign-up a shopper to a loyalty program.
  - [Card Acquisition requests followed by a payment](https://docs.adyen.com/point-of-sale/card-acquisition/#continue-with-payment)
  - [Card Acquisition requests followed by a cancellation](https://docs.adyen.com/point-of-sale/card-acquisition/#cancel-completed)

* Added an Admin-panel that shows a list of loyalty-enrolled shoppers

**The basic flow looks as follows:**
1. Make a card acquisition request
2. Present your card on the terminal 
3. Terminal asks the shopper whether they want to sign-up for a loyalty program (collect +100 points on every purchase, and get a discount after 200 points)
4. Terminal asks the shopper to:
  - (1) Finish the the request with a payment: shopper pays and is signed up for the loyalty program if they consent.
  - (2) Finish the request with a cancellation request: shopper only signs up for the loyalty program (without making a payment).

The demo features both (1) and (2)  -> selectable from the main index.html page.

### CODE REVIEW

File: in-person-payments-loyalty-example/readme.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+﻿# Adyen [In-person Payment Loyalty](https://docs.adyen.com/point-of-sale/) Integration Demo
+
+This repository features an in-person payments [cloud terminal API](https://docs.adyen.com/point-of-sale/design-your-integration/choose-your-architecture/cloud/) loyalty example. 
+This enables you to check cardholder details before applying discounts or sign-up a shopper to a loyalty program.
+- [Card Acquisition requests followed by a payment](https://docs.adyen.com/point-of-sale/card-acquisition/#continue-with-payment)
+- [Card Acquisition requests followed by a cancellation](https://docs.adyen.com/point-of-sale/card-acquisition/#cancel-completed)
+
+The basic flow is as follows:
+1. Make a card acquisition request
```
Comment (gcatanese) [inline in-person-payments-loyalty-example/readme.md:9]: Maybe the basic flow should mention the purchase of a pizza as starting point

**Review by gcatanese — COMMENTED** [review]
Good work here. We should also add the example to `.gitpod.yml`

File: readme.md [hunk 2]
```diff
@@ -58,6 +59,15 @@ The [in-person payments example](in-person-payments-example) features an in-pers
 
 ![Card In-person Payments Demo](in-person-payments-example/wwwroot/images/cardinpersonpayments.gif)
 
+## [In-person Payments Loyalty Example](in-person-payments-loyalty-example)
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/in-person-payments-loyalty-example)
+
+[First time with Gitpod?](https://github.com/adyen-examples/.github/blob/main/pages/gitpod-get-started.md)
+
+The [in-person payments loyalty example](in-person-payments-loyalty-example) features an in-person payment [cloud terminal API](https://docs.adyen.com/point-of-sale/design-your-integration/choose-your-architecture/cloud/) integration. Within this demo app, you can make in-person payments using a terminal, create card acquisition requests, collect card details and apply discounts.
+
```
[the remark this answers is not in the document]
Comment (Kwok-he-Chu) [reply readme.md:68]: Fixed

File: in-person-payments-loyalty-example/readme.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+﻿# Adyen [In-person Payment Loyalty](https://docs.adyen.com/point-of-sale/) Integration Demo
+
+This repository features an in-person payments [cloud terminal API](https://docs.adyen.com/point-of-sale/design-your-integration/choose-your-architecture/cloud/) loyalty example. 
+This enables you to check cardholder details before applying discounts or sign-up a shopper to a loyalty program.
+- [Card Acquisition requests followed by a payment](https://docs.adyen.com/point-of-sale/card-acquisition/#continue-with-payment)
+- [Card Acquisition requests followed by a cancellation](https://docs.adyen.com/point-of-sale/card-acquisition/#cancel-completed)
```
[the remark this answers is not in the document]
Comment (Kwok-he-Chu) [reply in-person-payments-loyalty-example/readme.md:6]: Fixed, we'll have to do this for every repo 👍 (or as we-go)

**Pull request merged**